src/main.js

```diff
@@ -0,0 +1,87 @@
+const myLibrary = [];
+
+function Book(author, title, pages, read) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+function setStatus(read, button) {
+  if (read) {
+    button.innerText = 'Read';
+  } else {
+    button.innerText = 'Not read yet';
+  }
+}
+
+function switchStatus(e) {
+  const index = e.srcElement.attributes['1'].value;
+  myLibrary[parseInt(index, 10)].read = !myLibrary[parseInt(index, 10)].read;
+  setStatus(myLibrary[parseInt(index, 10)].read, e.target);
```

Comment:
If you do the `parseInt` once when you get the variable you can clean all of this a little bit:

```js
  const index = parseInt(e.target.getAttribute("rel"), 10);
  myLibrary[index].read = !myLibrary[index].read;
  setStatus(myLibrary[index].read, e.target);
```